src/utils.py

```diff
@@ -706,20 +734,27 @@ def filter_static_operators(ground_ops: Collection[GroundNSRTOrSTRIPSOperator],
                for op in ground_ops):
             continue
         static_preds.add(pred)
-    static_facts = {atom for atom in atoms if atom.predicate in static_preds}
+    static_atoms = {atom for atom in atoms if atom.predicate in static_preds}
+    return static_preds, static_atoms
+
+
+def filter_static_operators(ground_ops: Collection[GroundNSRTOrSTRIPSOperator],
```

Comment:
These are different from the reachable_nsrts stuff you added, right? Are you clear what the relationship between these things is?

Reply:
Are the operators that would get filtered out by static atoms check always a subset of the operators that would get filtered out by reachable set?

Reply:
yes, so I will remove `filter_static_operators`